packages/trader/src/Assets/Trading/Categories/Description/multiplier-trade-description.tsx

```diff
@@ -0,0 +1,102 @@
+            </Text>
+            <ul>
+                <Text as='li'>
+                    <Localize
+                        i18n_default_text={
+                            'If you select “Take profit” and specify an amount that you’d like to earn, your position will be closed automatically when your profit is more than or equals to this amount. Your profit may be more than the amount you entered depending on the market price at closing.'
+                        }
+                    />
+                </Text>
+                <Text as='li'>
+                    <Localize
+                        i18n_default_text={
+                            'If you select “Stop loss” and specify an amount to limit your loss, your position will be closed automatically when your loss is more than or equals to this amount. Your loss may be more than the amount you entered depending on the market price at closing.'
+                        }
+                    />
+                </Text>
+                <Text as='li'>
+                    <Localize
+                        i18n_default_text={
+                            'If you select “Deal cancellation”, you’ll be able to cancel your trade within a chosen time frame should the market move against your favour. We’ll charge a small fee for this, but we’ll return your stake amount without profit or loss. If the stop-out amount is reached before the deal cancellation expires, your position will be cancelled automatically and we’ll return your stake amount without profit or loss.'
+                        }
+                    />
+                </Text>
+                <Text as='p'>
+                    <Localize i18n_default_text={'While “Deal cancellation” is active:'} />
+                </Text>
+                <ul>
+                    <Text as='li'>
+                        <Localize
+                            i18n_default_text={
+                                '“Stop loss” is deactivated and will only be available when “Deal cancellation” expires.'
+                            }
+                        />
+                    </Text>
+                    <Text as='li'>
+                        <Localize
+                            i18n_default_text={
+                                '“Take profit” cannot be updated. You may update it only when “Deal cancellation” expires.'
+                            }
+                        />
+                    </Text>
+                </ul>
+            </ul>
+            <Text as='p'>
+                <Localize
+                    i18n_default_text={
+                        'The entry spot is the market price when your contract is processed by our servers.'
+                    }
+                />
+            </Text>
+            <Text as='p'>
+                <Localize i18n_default_text={'The exit spot is the market price when the contract is closed.'} />
+            </Text>
+        </React.Fragment>
+    );
+};
+
```

Comment:
I also tried approach from other files (store all `<Localize />` in array with objects and parse them with the help of `.map `method), but the array with objects become too much multileveled, so I decided to leave file in the same condition as before refactoring.
Example of another approach (regected by me):
```
const content = [
        {
            type: 'paragraph',
            text: (
                <Localize
                    i18n_default_text={
                        'Predict the market direction and select either “Up” or “Down” to open a position. We will charge a commission when you open a position.'
                    }
                />
            ),
        },
        {
            type: 'list',
            text: [
                {
                    type: 'list_item',
                    text: (
                        <Localize
                            i18n_default_text={
                                'If you select “Take profit” and specify an amount that you’d like to earn, your position will be closed automatically when your profit is more than or equals to this amount. Your profit may be more than the amount you entered depending on the market price at closing.'
                            }
                        />
                    ),
                },
                {
                    type: 'list',
                    text: [
                        {
                            type: 'list_item',
                            text: (
                                <Localize
                                    i18n_default_text={
                                        'If you select “Take profit” and specify an amount that you’d like to earn, your position will be closed automatically when your profit is more than or equals to this amount. Your profit may be more than the amount you entered depending on the market price at closing.'
                                    }
                                />
                            ),
                        },
                    ],
                },
            ],
        },
    ] as const;
    const parser = (array: Partial<typeof content>) => {
        return array.map(({ type, text }, index) => {
            if (type === 'paragraph') {
                return (
                    <Text as='p' key={index.toString() + text}>
                        {text}
                    </Text>
                );
            }
            if (type === 'list_item') {
                return (
                    <Text as='li' key={index.toString() + text}>
                        {text}
                    </Text>
                );
            }
            return <ul key={index.toString()}>{parser(text)}</ul>;
        });
    };
    console.log('test', parser(content));
    return <React.Fragment>{parser(content)}</React.Fragment>;

```

Reply:
@kate-deriv no need to create 1 multi-leveled object, you can create 4 arrays with content, e.g. array #1 for the first block of paragraphs, array #2 for the block of li's inside the 1st `ul`, array #3 for the block of li's inside the 2nd `ul` (or array #2 and array #3 can be united into an object), array #4 for the last 2 paragraphs. naming is up to you but it's better to make it semantic, e.g.  about_section / multipliers_params_section / deal_cancellation_section / closing_section